pages/settings.js

```diff
@@ -85,7 +80,7 @@ export default function SettingForm() {
           helpText={<span>Apply this tag to the order</span>}
         />
 
-        <Button primary disabled={!submitable} loading={loading} submit>
+        <Button primary disabled={!(price && tag)} loading={loading} submit>
```

Comment:
should use a submitable and set it to !(price && tag), it will make the code more readable